integration/docker/dockerfile-common.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+#
+# The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+# (the "License"). You may not use this work except in compliance with the License, which is
+# available at www.apache.org/licenses/LICENSE-2.0
+#
+# This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied, as more fully set forth in the License.
+#
+# See the NOTICE file distributed with this work for information regarding copyright ownership.
+#
+
+function installLibfuse {
+  git clone https://github.com/cheyang/libfuse.git
+  cd libfuse
+  git checkout fuse_2_9_5_customize_multi_threads_no_logging
+  bash makeconf.sh
+  ./configure
+  make -j8
+  make install
+  cd ..
```

Comment:
emmmm.... not sure how bash deal with locations,, cd libfuse and cd .. are a little strange

Reply:
Good question. They do work in my tests (I can see Fuse being installed when build the image and I can successfully mount with Fuse and see the files in worker container) so I didn't really pay attention. Seems like normally they don't work in bash scripts so I honestly don't know why they work here - maybe because it's called in a Dockerfile?